app/js/controllers/copy_move.js

```diff
@@ -26,9 +26,13 @@ export default class MoveController extends Controller {
     this.view.dialog.open();
   }
 
+  setMode(mode) {
```

Comment:
This is pretty vague. I'd rename it to ```setCopyOrMoveMode()```, or something like that, and add a comment explaining it.